abcpy/ProbabilisticModel.py

```diff
@@ -6,164 +6,289 @@
 
 #TODO both ricker and lorenz could support that you give some in a combined model, and some not, we should implement that, but not a priority
 
-#TODO docstrings
+#NOTE we could call self.parents self.prior?
 
 #NOTE do we maybe want to average over a couple of samples during initialization, rather than taking a single value? this is an issue for example for StudentT!
```

Comment:
I noticed that for example for StudentT, the sampled values, depending on the defined parameters, will vary a lot (simply because the variance is large). We right now initialize parameter values coming from other probabilistic models by sampling them. If we sample from a distribution with a large variance, we might not get a good representation of that parameter value. Would it maybe make sense to at initialization average over multiple samples? I am not sure whether this would also make sense when we sample from the prior in general.